internal/cli/atlas/clusters/create.go

```diff
@@ -244,6 +268,9 @@ For full control of your deployment, or to create multi-cloud clusters, provide
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return opts.Run()
 		},
+		PostRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PostRun()
+		},
```

Comment:
@colm-quinn @gssbzn 
The e2e for Cluster [create](https://spruce.mongodb.com/task/mongodb_atlas_cli_master_e2e_atlas_clusters_atlas_clusters_flags_e2e_patch_748e1c401590c48af9280e235243698fac808af7_648739499ccd4e678a22f290_23_06_12_15_27_06/tests?execution=0&sortBy=STATUS&sortDir=ASC) and delete are failing now because the response for the commands has changed. 

Previously, `clusters create` used to return a clusters object when successful or error. Now, with the watch command being run as `PostRunE` function, we get the response as Nil when the watch is a success or error.

I would like to know what is the behaviour and response we expect for the commands?

Reply:
I would expect `atlas clusters create <name>` or `atlas clusters create <name> --watch` should return the same output, ie
```
Cluster <name> created successfully
```

Why?
```bash
atlas clusters create MyDemo --watch -o json-path=.SrvConnection | mongosh
```
The above should work (I don't rememebr the field of the srv url so that is probably wrong)

Reply:
> The e2e for Cluster are failing now because the response for the commands has changed.
 
If there is changed response in existing feature (non-watch) we need to flag that somehow